nx-X11/config/cf/Imake.cf

```diff
@@ -841,6 +841,12 @@ XCOMM Keep cpp from replacing path elements containing i486/i586/i686
 #  define Mc68020Architecture
```

Comment:
> Doesn't this revert f6dcf2f partially? It doesn't bring back support for HP-UX, but pulls in back some parts for HPPA. It also seems to be missing some hunks.

Yes, it actually does. Good point. I have now picked the __hppa__ related stuff back in while leaving the hpux stuff out. The commit now brings back the old code, so things should look similar as compared to the earlier state.